agera/src/main/java/com/google/android/agera/ScheduledUpdatables.java

```diff
@@ -4,21 +4,12 @@
 
 import java.util.Arrays;
 
-final class IdentityMultiMap<K, V> {
+final class ScheduledUpdatables<K, V> {
```

Comment:
Well....... this map when used in BaseObs isn't for the 'scheduled' updatables.

You can call this UpdatableMultimap though; and forget the type parameter K -- it's always Updatable.